torch/fx/experimental/symbolic_shapes.py

```diff
@@ -530,15 +545,20 @@ def create_symintnode(self, sym: "sympy.Expr"):
     # This is guaranteed to return a symbol or its negation is a sympy.Symbol,
     # but there may be a replacement that allows it to be immediately
     # simplified
-    def create_symbol(self, val: int, *, simplify: bool = True) -> "sympy.Expr":
+    def create_symbol(self, val: int, *, simplify: bool = True, sname: str) -> "sympy.Expr":
+        assert isinstance(sname, str), f"{type(sname)} {sname}"
```

Comment:
nit: type is enough probably, if `sname` is a tensor this print is gonna be ugly